script.js

```diff
@@ -9,3 +9,17 @@
 // Add a `facial-hair` property to `instructor`.
 
 // Log the value of the `facial-hair` property to the console.
+
+var colors = ["red", 'blue', 'green']
+
+function hey () {
+for (var i=0; i < colors.length; i++)
+console.log("My favorite color is " + colors.if)
+}
+
+function instructor (facial-hair,name,hair-color){
+  this.facial-hair= beard;
+  this.name= james;
+  this.hair-color= brown;
+  console.log(instructor.facial-hair);
+}
```

Comment:
while this is close to how you might write a constructor function, there are a couple of issues here. first, that keys containing symbols require square bracket notation and not dot notation. `instructor.hair-color` is the same as `instructor.hair - color` but not `instructor['hair-color']` (correct). you would also have to use an object literal rather than a constructor for related reasons (invalid param name containing `-`)

I also see another issue-- `beard`, `james`, and `brown` are all references and not strings